cyences_app_for_splunk/default/app.conf

```diff
@@ -12,7 +12,7 @@ label = Cyences App for Splunk
 [launcher]
 author = CrossRealms International
 description = Cyences App for Splunk built by CrossRealms International
-version = 4.2.1
+version = 4.3.0
```

Comment:
Change version number in JS project for configuration page as well.

Reply:
Didn't get it? do you mean on cyences settings page anywhere? greped it but do not found.

Reply:
https://github.com/CrossRealms/Splunk-Cyences-App-for-Splunk/blob/master/product_setup_page/package.json

Reply:
Discussed with mahir, it is not required as we are not using it anywhere.